client/src/components/Navbar.js

```diff
@@ -19,16 +34,16 @@ function Navbar() {
           <nav>
             <ul>
               <li className="nav-item">
-                <a href="/home">HOME</a>
+                <button onClick={changeHome}>HOME</button>
               </li>
```

Comment:
Is there a reason why you didn't use the `Link` component that comes with `react-router-dom` but a button instead?

For example `<Link to="/example">EXAMPLE</Link>`

According to docs, I think this way you wouldn't need to use `nav()` and have those additional functions and the page should route automatically (For the sake of clarity the `Link` goes into an `li`)

Reply:
I did the buttons for simple css design. I can make it links and do a little touch up on it.

Reply:
Great, let me know if there are any blockers 👍